cogs/audio.py

```diff
@@ -942,6 +942,24 @@ def _stop_player(self, server):
             else:
                 await self._remove_song_status()
 
+    # no return. they can check themselves.
+    async def _notify_song_name(self):
+        if self.settings.get("NOTIFY"):
```

Comment:
This should really be a per server configurable option so all servers can have their song announced on change.